geometry_msgs/PoseStamped pose

float32 timeout_safety_factor
float32 angle_tolerance
float32 position_tolerance
float32 min_linear_speed
float32 max_linear_speed
float32 min_angular_speed
float32 max_angular_speed
float32 zero_epsilon
float32 stabilization_timeout
float32 steer_kP
float32 fine_steer_kP
float32 linear_kP
float32 timeout_turn_fudge
float32 timeout_fudge
float32 loopback_y_tolerance
float32 loopback_theta_tolerance
bool backwards_motion_only
bool turn_towards_final_heading
---
string status
---
